src/libasr/asdl_cpp.py

```diff
@@ -2529,6 +2559,9 @@ def main(argv):
         subs["MOD"] = "LPython::AST"
         subs["mod"] = "ast"
         subs["lcompiler"] = "lpython"
+    elif subs["MOD"] == "AST":
```

Comment:
Shouldn't it be `"LFORTRAN"` instead of `"AST"`?

Reply:
We haven't been consistent in the asdl if I remember well, so I would not do any big changes in the sync PRs, but we can fix things up later.